src/main/java/htsjdk/samtools/util/AbstractProgressLogger.java

```diff
@@ -60,10 +61,18 @@ private synchronized void record() {
         if (this.lastChrom == null) readInfo = "*/*";
         else readInfo = this.lastChrom + ":" + fmt.format(this.lastPos);
 
+        final String rnInfo;
+        if(lastReadName != null) {
+            rnInfo = ".  Last read name: " + lastReadName;
```

Comment:
I would like this to say "record" instead of "read", to be able to log non-reads record names.